kola-denylist.yaml

```diff
@@ -62,3 +62,20 @@
   osversion:
     - c9s
   snooze: 2022-11-15
+
+- pattern: ext.config.shared.networking.force-persist-ip
+  tracker: https://github.com/fedora-selinux/selinux-policy/commit/c86d943538f907c2e6b20ffda0a8d2b5b5bd2e34
+  osversion:
+    - rhel-9.0
```

Comment:
Let's snooze them until e.g. mid-January? Snoozing isn't the best UX, but it's a crude way to ensure we don't keep disabling those tests well past when they're no longer needed.

Reply:
I don't think the tests will ever pass in rhel 9.0.  Note the dot-zero.

Reply:
In that case, I think ideally these tests should be skipped on RHEL 9.0 entirely. We have `distros`, but I don't think we can limit per version yet, but it could make sense to add.

Reply:
Agree, though in practice what I think will happen is that we branch and main/master retargets for 9.2 at some point where this is already fixed.

Another way to say this is - the tests should automatically be un-denied when we retarget, which is the main goal.